src/EvoSC.Commands/Interfaces/IParserResult.cs

```diff
@@ -19,4 +19,12 @@ public interface IParserResult
     /// this will be set. But if Success is true, this may or may not be set.
     /// </summary>
     public Exception Exception { get; }
+    /// <summary>
+    /// Whether this command was intended. Essentially meaning the user used the command prefix.
```

Comment:
Was an attempt at a very short explanation. You either have what I call an "intended" or "not intended" execution of a command. You can imagine, if a user types the command prefix "/", in 99% of the cases, the user's intent is to execute some type of command. So in that case, the command execution is *intended*. However, aliases typically don't require this prefix, and most normal messages will look like aliases to the application, but they are most likely *not intended* as a command. This variable tells the application whether the user deliberately intended to execute a command or not. The main reason for this thing is for error reporting so that we can respond with errors to the user.

Reply:
I tried to rephrase it, but its difficult without writing a huge comment explaining what it is.